src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <img src="https://github.com/hserpukhovitina/layout_moyo-header/blob/master/src/images/logo.png?raw=true" alt="Moyo">
+      <nav class="nav">
+        <a class="nav-link is-active" href="#">APPLE</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">apple</a>
```
Семантично правильно використовувати таку структуру для навігації
Також, тексти пунктів пиши в нижньому регістрі, а великими буквами роби їх в css